File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.py
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        p_d, s_d = 1, 0
        
        while n > 0:
            m = n % 10
            p_d *= m
            s_d += m
            n = n // 10
            
        return p_d - s_d

File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.go
func subtractProductAndSum(n int) int {
    p_d := 1
    s_d := 0
    
    for n > 0 {
        m := n % 10
        p_d *= m
        s_d += m
        n = n / 10
    }
    return p_d - s_d
}